Sources/AirRobeWidget/Service/Endpoints/AirRobeWidgetEndpoints.swift

```diff
@@ -58,4 +58,20 @@ extension AirRobeEndpoint {
         )
     }
 
+    static func identifyOrder(orderId: String) -> AirRobeEndpoint {
+        let requestBody: [String: Any] = [
+            "app_id": configuration?.appId ?? "",
+            "anonymous_id": UIDevice.current.identifierForVendor?.uuidString ?? "",
+            "session_id": sessionId,
+            "external_order_id": orderId,
+            "split_test_variant": "default",
+            "opted_in": UserDefaults.standard.OrderOptedIn
```

Comment:
I don't think `UserDefaults.standard.OrderOptedIn` should be stored as a constant. I think we should pass it in as an argument with `orderId`.

Reply:
Ok so like this?
```swift
static func identifyOrder(orderId: String, orderOptedIn: Bool) -> AirRobeEndpoint
```

And call like this 👇 
```swift
apiService.identifyOrder(orderId: orderId, orderOptedIn: UserDefaults.standard.OrderOptedIn)
```